lib/internal/test_runner/utils.js

```diff
@@ -240,6 +241,9 @@ function parseCommandLine() {
         testNamePatternFlag,
         (re) => convertStringToRegExp(re, '--test-name-pattern'),
       ) : null;
+    const testSkipPatternFlag = getOptionValue('--test-skip-pattern');
+    testSkipPatterns = testSkipPatternFlag?.length > 0 ?
+      ArrayPrototypeMap(testSkipPatternFlag, (re) => convertStringToRegExp(re, '--test-skip-pattern')) : null;
```

Comment:
```suggestion
    const testSkipPatternFlag = getOptionValue('--test-skip-pattern');
    if (testNamePatternFlag?.length > 0) {
  	  testNamePatterns =   ArrayPrototypeMap(testNamePatternFlag,
      ArrayPrototypeMap(testNamePatternFlag, (re) => convertStringToRegExp(re, '--test-name-pattern)))
    } elseif if (testSkipPatternFlag?.length > 0) {
  	  testNamePatterns =   ArrayPrototypeMap(testSkipPatternFlag,
      ArrayPrototypeMap(testSkipPatternFlag, (re) => convertStringToRegExp(`(?!(${re}))`, '--test-skip-pattern')))
    } 
```

I don't think the implementation should contain much more than this^.

Reply:
I don't see an issue with the current implementation, as it's how Node has done it in the past. Is there a reason for this suggested change?